historical_system_profiles/views/v1.py

```diff
@@ -197,14 +234,17 @@ def ensure_rbac_hsps_read():
     # permissions=[["drift:*:*"], ["drift:notifications:read", "drift:baselines:read"]]
     # If we just have *:*, it works, but if not, we need both notifications:read and
     # baselines:read in order to allow access.
+    if g.get("rbac_filters") is None:
+        g.rbac_filters = {}
     return view_helpers.ensure_has_permission(
         permissions=[["drift:*:*"], ["drift:historical-system-profiles:read"]],
         application="drift",
-        app_name="historical-system-profiles",
+        app_name="historical-system-profiles,inventory",
```

Comment:
This should not be here. I know I told you to put it here but that was wrong info.
See https://github.com/RedHatInsights/system-baseline-backend/pull/535 and https://issues.redhat.com/browse/RHINENG-5551 for more info.